plugins/woocommerce-admin/client/tasks/hooks/useActiveSetupList.ts

```diff
@@ -8,15 +8,7 @@ export const useActiveSetupTasklist = () => {
 	const { activeSetuplist } = useSelect( ( select ) => {
 		const taskLists = select( ONBOARDING_STORE_NAME ).getTaskLists();
 
-		const visibleSetupList = taskLists
-			.filter( ( list ) => list.isVisible )
-			.filter( ( list ) =>
-				[
-					'setup_experiment_1',
-					'setup_experiment_2',
-					'setup',
-				].includes( list.id )
-			);
+		const visibleSetupList = taskLists.filter( ( list ) => list.isVisible );
```

Comment:
I think we still need to filter the id here because there are other task lists such as "extended" or "secret_tasklist".

```suggestion
		const visibleSetupList = taskLists.filter(
			( list ) => list.id === 'setup' && list.isVisible
		);
```

It only leads to a UI bug when the setup task is dismissed. 

![Screen Shot 2022-07-28 at 12 34 44](https://user-images.githubusercontent.com/4344253/181421456-901a337d-8a6a-4a67-841e-0090dd1d9257.png)

Reply:
@chihsuan Thank you for the suggestion 👍 Committed